.github/workflows/kind.yml

```diff
@@ -77,16 +77,8 @@ jobs:
     - uses: actions/setup-go@v2
       with:
         go-version: 1.17
-    - name: Download Antrea image from previous job
-      uses: actions/download-artifact@v1
-      with:
-        name: antrea-ubuntu-cov
-      # TODO: Create path to two image artifacts when uploading artifacts, so multiple
-      # artifacts can be downloaded at once.
-    - name: Download Flow Aggregator image from previous job
-      uses: actions/download-artifact@v1
-      with:
-        name: flow-aggregator-cov
+    - name: Download Antrea images from previous job
```

Comment:
nit: I guess it should be "previous jobs" since there are multiple jobs building & uploading these artifacts

Reply:
Sure.